src/Elcodi/Plugin/DisqusBundle/Resources/config/plugin.yml

```diff
@@ -0,0 +1,9 @@
+services:
+
+    #
+    # Plugin
+    #
+    elcodi_plugin.disqus:
```

Comment:
Wouldn't be easier to generate this services automatically via `CompilerPass`?
The less burden for plugin creators, the better.